server/routes/reports/prisoners/involvement/controller.ts

```diff
@@ -0,0 +1,75 @@
+import type express from 'express'
+import type FormWizard from 'hmpo-form-wizard'
+
+import type { PrisonerInvolvementOutcome, PrisonerInvolvementRole } from '../../../../reportConfiguration/constants'
+import { BaseController } from '../../../../controllers'
+import { convertToTitleCase, nameOfPerson, possessive } from '../../../../utils/utils'
+import type { ReportWithDetails } from '../../../../data/incidentReportingApi'
+import type { Values } from './fields'
+
+// eslint-disable-next-line import/prefer-default-export
+export abstract class PrisonerInvolvementController extends BaseController<Values> {
+  middlewareLocals(): void {
+    this.use(this.customiseFieldLabels)
+    super.middlewareLocals()
+  }
+
+  customiseFieldLabels(req: FormWizard.Request<Values>, res: express.Response, next: express.NextFunction): void {
+    const { firstName } = this.getPrisonerName(res)
+    const possessiveFirstName = possessive(convertToTitleCase(firstName))
+
+    const { fields: customisedFields } = req.form.options
+    customisedFields.prisonerRole = {
+      ...customisedFields.prisonerRole,
+      label: `What was ${possessiveFirstName} role?`,
+    }
+    customisedFields.comment = {
+      ...customisedFields.comment,
+      label: `Details of ${possessiveFirstName} involvement (optional)`,
+    }
+
+    next()
+  }
+
+  locals(req: FormWizard.Request<Values>, res: express.Response): Partial<FormWizard.Locals<Values>> {
+    return {
+      ...super.locals(req, res),
+      pageTitle: `${possessive(nameOfPerson(this.getPrisonerName(res)))} involvement in the incident`,
+    }
+  }
+
+  protected errorMessage(error: FormWizard.Error): string {
+    if (error.key === 'prisonerRole') {
+      return 'Chose the prisoner’s role'
```

Comment:
Shouldn't this be 'choose' (two 'o's)? (same below for the outcome field)

Reply:
Ops, I now realise it was fixed in a later commit, soz

Reply:
fixed in a later commit!